frontend/tests/e2e_tests/docker-compose.e2e-tests.yml

```diff
@@ -33,18 +35,43 @@ services:
       - ${GUI_REPOSITORY}/tests/e2e_tests/utils:/e2e/utils
       - ${GUI_REPOSITORY}/videos:/e2e/videos
 
-  mender-client:
-    image: mendersoftware/mender-client-docker-addons:mender-3.6.x
+  client:
+    image: mendersoftware/mender-client-docker-addons:mender-master
     volumes:
       - ${GUI_REPOSITORY}/tests/e2e_tests/dockerClient/mender.json:/etc/mender/mender.conf
       - ${GUI_REPOSITORY}/tests/e2e_tests/dockerClient/mender-connect.json:/etc/mender/mender-connect.conf
 
+  create-artifact-worker:
+    image: mendersoftware/create-artifact-worker:main
+  deployments:
+    image: mendersoftware/deployments:main
+  deviceauth:
+    image: mendersoftware/deviceauth:main
+  deviceconfig:
+    image: mendersoftware/deviceconfig:main
+  deviceconnect:
+    image: mendersoftware/deviceconnect:main
+  inventory:
+    image: mendersoftware/inventory:main
+  iot-manager:
+    image: mendersoftware/iot-manager:main
+  useradm:
+    image: mendersoftware/useradm:main
+  workflows:
+    image: mendersoftware/workflows:main
+  workflows-worker:
+    image: mendersoftware/workflows:main
```

Comment:
Just one thing I forgot last round.
Have you considered setting the environment variables?
```bash
MENDER_IMAGE_PREFIX="docker.io/mendersoftware"
MENDER_IMAGE_TAG="${CI_COMMIT_TAG:-main}"
```